covalent/_shared_files/defaults.py

```diff
@@ -73,6 +73,7 @@
     "user_interface": {
         "address": "localhost",
         "port": 48008,
+        "dev_port": 49009,
```

Comment:
@wjcunningham7 Since we are using the port 49009 as a development port for the UI, that port should be allowed in the cors-origin. To do that, we have added 49009 as dev_port in defaults.py to avoid hard-coding it on the cors-origin. Please let us know your thought on this change.

Reply:
@Aravind-psiog if the value for `dev_port` is to correspond to `port` for production please just use the `port` value from the config and add a environment variable that can override the `port` variable in `covalent_ui/api/main.py`

Reply:
Hi @AlejandroEsquivel - This is a port used when a developer runs the front end stand alone using `yarn run start`. This is not used in production nor it is used when `app.py` is run